src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/sdk/__init__.py

```diff
@@ -0,0 +1,17 @@
+# coding=utf-8
```

Comment:
Nope. Will not merge in until azure-mgmt-monitor is released. PR https://github.com/Azure/azure-sdk-for-python/pull/974